packages/react-components/react-tree/src/components/TreeItemAside/renderTreeItemAside.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+import { createElement } from '@fluentui/react-jsx-runtime';
+import { getSlotsNext } from '@fluentui/react-utilities';
+import { ButtonContextProvider } from '@fluentui/react-button';
+import type { TreeItemAsideState, TreeItemAsideSlots } from './TreeItemAside.types';
+
+/**
+ * Render the final JSX of TreeItemAside
+ */
+export const renderTreeItemAside_unstable = (state: TreeItemAsideState) => {
+  const { slots, slotProps } = getSlotsNext<TreeItemAsideSlots>(state);
+
+  if (!state.visible) {
```

Comment:
Also when the mouse leaves the menu, the entire actions is no longer rendered which results in the menu being dismissed. It's not a big deal but just wanted to point it out, because this is not what the Teams chat list does
![actions menu dismiss](https://github.com/microsoft/fluentui/assets/20744592/d8333652-dd79-4271-841e-bb0790e3d15c)